core/src/main/java/org/mule/runtime/core/privileged/exception/TemplateOnErrorHandler.java

```diff
@@ -494,37 +508,41 @@ public void setFlowLocation(Location location) {
    */
   public abstract TemplateOnErrorHandler duplicateFor(Location location);
 
-
   private boolean isTransactionInGlobalErrorHandler(TransactionAdapter transaction) {
     String transactionContainerName = transaction.getComponentLocation().get().getRootContainerName();
     return flowLocation.isPresent() && transactionContainerName.equals(flowLocation.get().getGlobalName());
   }
 
-  protected boolean isOwnedTransaction() {
+  protected boolean isOwnedTransaction(Exception exception) {
```

Comment:
this is privileged, will have to keep a method with the previous signature just in case